utilities/trajectory_generator.py

```diff
@@ -0,0 +1,47 @@
+from wpimath.geometry import Pose2d, Translation2d, Rotation2d
+from typing import List
+
+from utilities.scalers import scale_value
+
+
+def totalLength(waypoints: List[Pose2d]) -> float:
+    return sum(
+        last.translation().distance(cur.translation())
+        for last, cur in zip(waypoints, waypoints[1:])
+    )
+
+
+def lerpPose(p1: Pose2d, p2: Pose2d, n1: float) -> Pose2d:
+    n2 = 1 - n1
+    translation = Translation2d(
+        x=p1.translation().x * n2 + p2.translation().x * n1,
+        y=p1.translation().y * n2 + p2.translation().y * n1,
+    )
```

Comment:
You should be able to scale (scalar multiplication) and translate (add) `Translation2d`s, right?